lib/livebook_web/live/session_live.ex

```diff
@@ -1951,4 +2001,12 @@ defmodule LivebookWeb.SessionLive do
 
     :ok
   end
+
+  defp session_only_secrets(secrets, livebook_secrets) do
+    Enum.reject(secrets, &(&1 in livebook_secrets)) |> Enum.sort()
```

Comment:
```suggestion
    Enum.sort(secrets -- livebook_secrets)
```

Reply:
This won't work I think, since they are maps. :D

Reply:
For large lists I think the current version is faster because `in` will lookup the value in the map and compare it. :)

Reply:
Isn't `{key, value}` lookup linear, or is it optimised?

Reply:
Optimized: https://github.com/elixir-lang/elixir/blob/main/lib/elixir/lib/enum.ex#L4759

Reply:
Beautiful!! Thank you :heart: